app/scrapers/bing.py

```diff
@@ -30,3 +34,39 @@ def parse_response(self, soup):
         print('Bing parsed: ' + str(urls))
 
         return urls
+
+    def parse_video_response(self, soup):
+        """ Parse response and returns the urls
+
+            Returns: urls (list)
+                    [[Tile1, url1], [Title2, url2], ...]
+        """
+        urls = []
+        for a in soup.findAll('a', attrs={'class': 'mc_vtvc_link'}):
+            title = a.get('aria-label').split(' Duration')[0]
+            url = 'https://www.bing.com' + a.get('href')
+            urls.append({
+                'title': title,
+                'link': url
+            })
+
+        print('Bing parsed: ' + str(urls))
+
+        return urls
+
+    def parse_image_response(self, soup):
```

Comment:
This method can be changed to a static method by using `@staticmethod` and removing `self` from the position argument. Codacy also suggests the same modification.

Reply:
Since previously it was not useful so not using this time too.